.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -119,36 +104,22 @@ body:
     id: "nightwatch-version"
     attributes:
       label: "Nightwatch.js Version"
-      description: "The version of Nightwatch you use."
+      description: "The version of Nightwatch you use. `npx nightwatch --version`"
       placeholder: "2.3.0"
     validations:
       required: true
-  - type: "input"
-    id: "npm-version"
-    attributes:
-      label: "NPM Version"
-      description: "The version of NPM you use."
-      placeholder: "8.12.1"
-    validations:
-      required: true
```

Comment:
I think we need this
Cc @beatfactor

Reply:
NPM version should stay but can be optional.

Reply:
I think the form is already long enough.